utils.py

```diff
@@ -39,4 +39,9 @@ def get_ena_metadata_from_csv(sample_name, watch_dir):
         else:
             return None
 
-
+def is_number(string):
+    try:
+        float(string)
```

Comment:
`>>> float()
0.0
1. This code returns true if the method is called with an empty string. 
2. If it fails to convert to float, return False to any kind of exception unless you plan to give a specific message to different kinds of errors. For instance, a TypeError wouldn't  be managed 

`float(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: float() argument must be a string or a number, not 'NoneType`

Reply:
Adding checking for empty and blank strings
Normall you should never except without an error type, but this may be an exception